tests/conftest.py

```diff
@@ -54,7 +55,7 @@ def rule(rule_name: str, config: Config, rule_path: str) -> Optional[Rule]:
 
 @pytest.fixture
 def rule_path() -> str:
-    return "credsweeper/rules/config.yaml"
+    return str(CREDSWEEPER_DIR / "rules" / "config.yaml")
```

Comment:
os.path.join is the best here

Reply:
pathlib.Path returns not exactly string which might fail in python3.7 in windows.
https://github.com/Samsung/CredSweeper/runs/7797164059?check_suite_focus=true
e.g. 
```
tests\test_app.py:31: in _m_credsweeper
    stderr=subprocess.PIPE)  #
c:\hostedtoolcache\windows\python\3.7.9\x64\lib\subprocess.py:800: in __init__
    restore_signals, start_new_session)
c:\hostedtoolcache\windows\python\3.7.9\x64\lib\subprocess.py:1148: in _execute_child
    args = list2cmdline(args)
```